public/testem/testem_connection.js

```diff
@@ -175,7 +175,7 @@ function patchEmitterForWildcard(socket) {
 }
 
 function initSocket(id) {
-  socket = io.connect({ reconnectionDelayMax: 1000, randomizationFactor: 0 });
+  socket = io().connect({ reconnectionDelayMax: 1000, randomizationFactor: 0 });
```

Comment:
Given what the API is, this is actually redundant, it turns out. You can just call `io`, because [`io` *is* `connect`][wild].

```suggestion
  socket = io({ reconnectionDelayMax: 1000, randomizationFactor: 0 });
```

[wild]: https://github.com/socketio/socket.io-client/blob/c76d36778d337dc8aa9ef956e3624010e19bad94/lib/index.ts#L93-L94